DirectGuiDesigner/dialogs/AddItemDialog.py

```diff
@@ -95,8 +103,11 @@ def __init__(self, customWidget, child, childInfo, func, rootParent=None):
         if rootParent is None:
             rootParent = base.pixel2d
         super().__init__(rootParent)
+
+        self.value = {}  # use name to access the element selection for that parameter
+        self.radioList = {}  # dict[name: str, list[radiobutton]]
         self.labelList = []
-        self.entryList = []
+        self.entryList = {}
```

Comment:
If you change the type to dict, please also change the name to represent the change, so it's entryDict. That'll help to know what's in there.

Reply:
Yeah, I intended to change that, but I must have forgotten. Thank you for pointing that out